vendor/github.com/golang/protobuf/descriptor/descriptor_test.go

```diff
@@ -1,32 +0,0 @@
-package descriptor_test
```

Comment:
How come a lot of these vendor files were removed?

Reply:
Hm, I'll wait for the other PR with vendor changes to go in first, then merge and see if it shows up again.

Reply:
The vendor changes went in with the PR #268.